bigbluebutton/greenlight
# Accessibility fix for remove user access

Hi, This PR should enable keyboard user (user that visually impaired) to remove user access from the Share Room Access modal.
By using `tabs` key, the user will be able to get to `x` remove shared user.

## Description

It is just a small changes and all changes are on share room access modal:
1. (room.js) Change `x` (remove shared user) from <span> to <button>, so that it can be accessible using keyboards and have enter key and space key action by default
2. (locales) add remove_user so for aria-label, so screen reader can read it.


## Testing Steps

To test it:
1. Go to shared room access modal
2. If there is no user, add a user
3. User (you) should be able to remove the user that just added by using only keyboard, using tabs and enter / space key.

## Screenshots (if appropriate):

Not much changes, but if you look closely the `x` button is select able by keyboard now
![Screenshot from 2021-06-29 15-31-27](https://user-images.githubusercontent.com/29434066/123733963-8d576e00-d8f0-11eb-8fb8-e37b8c9ac92f.png)
